### ISSUE TITLE
Modules with long file paths can be published but not installed

### ISSUE THREAD

**Issue by michaeltlombardi** [puppetlabs/Puppet.Dsc#144]
## Describe the Bug

Using the functions in this module, it is possible to puppetize and publish a module with DSC Resources which cannot be installed via `puppet module install`.

## Expected Behavior

Modules published to the forge can be installed by end users.

## Steps to Reproduce

```powershell
# Validate the module is buildable
New-Item /tdsc -ItemType Directory
Import-Module -Name Puppet.Dsc
New-PuppetDscModule New-PuppetDscModule -PowerShellModuleName xremotedesktopsessionhost -Verbose -OutputDirectory /tdsc
# Show that the module built in this way is _not_ installable
puppet module install dsc-xremotedesktopsessionhost --version 2.0.0-0-1 --trace
```

## Additional Context

The root of this problem is with minitar as described in [PUP-10924](https://tickets.puppetlabs.com/browse/PUP-10924) and elaborated on in [PDK-1684](https://tickets.puppetlabs.com/browse/PDK-1684) - as long as we are unable to support long paths on module installation _even when the operating system supports those paths_ we will have modules which are not installable by end users.

This Issue needs to be resolved before a `1.0.0` release can be cut.

**Comment by michaeltlombardi** [comment]
Support for long file paths has [landed in core ruby](https://github.com/ruby/ruby/pull/4505) but not yet in minitar.

**Comment by michaeltlombardi** [comment]
Possibly resolved by #169 once testing and docs are implemented

**Comment by michaeltlombardi** [comment]
No longer able to reproduce the bug.
